modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -2,27 +2,43 @@
 //
 // * installing/installing_azure/installing-azure-user-infra.adoc
 // * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
+// * installing/installing_azure_stack_hub/installing-azure-stack-hub-default.adoc
 
 ifeval::["{context}" == "installing-azure-user-infra"]
 :azure:
 endif::[]
 ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
 :ash:
 endif::[]
+ifeval::["{context}" == "installing-azure-stack-hub-default"]
+:ash-ipi:
+endif::[]
 
 [id="installation-azure-user-infra-uploading-rhcos_{context}"]
+ifndef::ash-ipi[]
 = Uploading the {op-system} cluster image and bootstrap Ignition config file
+endif::ash-ipi[]
+ifdef::ash-ipi[]
+= Uploading the {op-system} cluster image
+endif::ash-ipi[]
+
+ifndef::ash-ipi[]
+The Azure client does not support deployments based on files existing locally. You
+must copy and store the {op-system} virtual hard disk (VHD) cluster image and bootstrap Ignition config file in a storage container so they are accessible during deployment.
+endif::ash-ipi[]
 
-The Azure client does not support deployments based on files existing locally;
-therefore, you must copy and store the {op-system} virtual hard disk (VHD)
-cluster image and bootstrap Ignition config file in a storage container so they
-are accessible during deployment.
+ifdef::ash-ipi[]
+The Azure client does not support deployments based on files existing locally. You
+must copy and store the {op-system} virtual hard disk (VHD) cluster image in a storage container so they are accessible during deployment.
```

Comment:
```suggestion
You
must copy and store the {op-system} virtual hard disk (VHD) cluster image in a storage container so they are accessible during deployment.
```
We don't use the `az` Azure client when running IPI with the installer. If we wanted to add a bit more explanation to this, we could say "Creating a virtual machine image requires a source VHD in the Azure Stack Hub environment." But I don't think that's strictly necessary.